modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/senna/utils/utils.js

```diff
@@ -229,6 +238,25 @@ class utils {
 			},
 		});
 	}
+
+	/**
+	 * Slices the given array, just like Array.prototype.slice, but this
+	 * is faster and working on all array-like objects (like arguments).
```

Comment:
I wonder if it is still faster in 2020... 🤔

Reply:
Maybe it wasn't even faster back then :D

Reply:
I didn't want to be mean, but yes...

![indeedd](https://user-images.githubusercontent.com/7074/98672276-dd7bc300-2355-11eb-9fd9-464da5d31e1c.gif)

Reply:
I think I got rid of this completely in my attempt to refactor utils at https://github.com/liferay-frontend/liferay-portal/pull/489/commits/8a020df1b4dd4bd5664b4dc170fcee69685b432d